app/utils/validators/forms/FieldValidator.scala

```diff
@@ -79,6 +79,8 @@ object FieldValidator {
 
   val isValidAddressField: String => Boolean = (input: String) => isAlphanumericWithSpecialCharacters(Set(' ', '\'', ',', '-'))(input)
 
+  val isValidMucr: String => Boolean = (input: String) => isAlphanumericWithSpecialCharacters(Set('\\', '-'))(input)
```

Comment:
This seems incorrect to me. For e.g. in Movements we validate MUCR against a regex: `GB/[0-9A-Z]{3,4}-[0-9A-Z]{5,28}|GB/[0-9A-Z]{9,12}-[0-9A-Z]{1,23}|A:[0-9A-Z]{3}[0-9]{8}|C:[A-Z]{3}[0-9A-Z]{3,30}`. Why is it different here?

Reply:
Wasn't aware. Ticket says nothing on this. Still, not sure the Movements regex is entirely correct (the colon IMO is wrong)